src/AzurePipelinesToGitHubActionsConverter.Core/Conversion/VariablesProcessing.cs

```diff
@@ -40,21 +42,26 @@ public VariablesProcessing(List<VariableGroup> variableGroups = null, bool verbo
         }
 
         //process all (simple) variables
-        public Dictionary<string, string> ProcessSimpleVariables(Dictionary<string, string> variables)
+        public OrderedDictionary ProcessSimpleVariables(OrderedDictionary variables)
         {
             if (variables != null)
             {
                 // update variables from the $(variableName) format to ${{variableName}} format, by piping them into a list for replacement later.
-                VariableList.AddRange(variables.Keys);
+                // VariableList.AddRange(variables.Keys);
```

Comment:
Let's remove this if we're not using it anymore?

Reply:
Yep